update-antora-spring-ui/action.yml

```diff
@@ -0,0 +1,88 @@
+name: Update Antora UI Spring
+description: A GitHub Action that detects if there is a new version of the Antora UI Spring artifact and creates a PR to update the antora-playbook.yml file to use the new version.
+
+inputs:
+  docs-branch:
+    description: The branch where the antora-playbook.yml file is. Default is docs-build
+    default: 'docs-build'
+  workflow-branch:
```

Comment:
Since there can be more than one PR branch (see comment on docs-branch), this probably needs to be based off of the docs-branch with a prefix or suffix. For example main_update-antora-ui-spring, 4.0.x_update-antora-ui-spring. Note that we have various rules in place if a branch ends with .x so starting with the branch name ensures we don't get problems with that.

Reply:
The branch name is now `${{ inputs.docs-branch }}_${{ inputs.workflow-branch-suffix }}`